aviary/docs/getting_started/onboarding_level1.ipynb

```diff
@@ -46,9 +46,10 @@
     "    Some variables are only used as initial guesses for the trajectory.\n",
     "    These are displayed separately from the Input Values because they will not be passed directly to components\n",
     "3. **Unconverted Values:**\n",
-    "    If the fortran_to_aviary converter can't find an Aviary variable with a `historical_name` that matches the FORTRAN variable, it is added to the end of the csv file.\n",
+    "    If the fortran_to_aviary converter can't find an Aviary variable that matches the FORTRAN variable, it is added to the end of the csv file.\n",
```

Comment:
I think this is beneficial to leave in, so users understand what is being "matched" to the Fortran variable.

Reply:
The paragraph before the list already described the matching with `historical_name`, so I was trying to cut down on repetitive information